### ISSUE TITLE
A "proper" browser/panel to your library of musical assets.

### ISSUE THREAD

**Issue by oldcastlehq** [rncbc/qtractor#302]
Hi,

A suggestion for a richer UX. The current File System feels like a "kitchen sink" solution. All the musicians that I know don't need their whole OS file system to be available to make their music. We usually organize our assets within folders. With that in mind, it would be great if we could add ONLY specific folders to the file system panel. The same way that Ableton Live does. So, we can have our assets more organised and easy to access them. Plus, a search function on this panel (only the files added to the file system would be indexed) would be fantastic. I have 5000+ samples and a search function for the file name (tags is asking too much, right? :-) ) is a life-saver.

**Comment by rncbc** [comment]
yep, that would be fantastic... but calling it "proper" is kind of a stretch...

the current File System widget remembers where you left it last time though and also what file-types were you interested in, whether audio or MIDI...

but ok... i do understand what you ask.

please, feel free to propose a PR. that would be even more fantastic! ;)

**Comment by oldcastlehq** [comment]
Sorry...about the "proper" word. I didn't mean to sound negative. My bad.
